options.go

```diff
@@ -178,6 +186,16 @@ func WithHTTPClient(client *http.Client) Option {
 func WithAdminAPIEndpoint(url string) Option {
 	return func(d *dialerConfig) {
 		d.sqladminOpts = append(d.sqladminOpts, apiopt.WithEndpoint(url))
+		d.serviceUniverseDomain = ""
```

Comment:
I'm not sure that custom endpoints follow any specific pattern do they? If they don't I'm not sure we can parse the universe domain from the URIs in a reliable manner...

Reply:
In second thought I guess the universe domain would be the suffix in format from the second last `.` to the end of the URI?

`https://sqladmin.googleapis.com` => `googleapis.com`

`https://sqladmin.apis-s3nscloud.io` => `apis-s3nscloud.io`

Reply:
Do the TPC folks have any guidance for parsing the universe from a URL? It would be nice to avoid setting this to empty string, which isn't semantically correct (although I see how you've got it working here).

Reply:
I assume this will be handled by the underlying APIARY sqladmin library when TPC changes happen in those in the near future